nbs/flags_page.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>"just" -&gt; "is just"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>